academic/scidavis/scidavis.info

```diff
@@ -1,10 +1,10 @@
 PRGNAM="scidavis"
-VERSION="1.26"
+VERSION="2.4.0"
 HOMEPAGE="http://scidavis.sourceforge.net/"
-DOWNLOAD="https://ufpr.dl.sourceforge.net/project/scidavis/SciDAVis/1.26/scidavis-1.26.tar.gz"
-MD5SUM="a3e6158c9607dbbaab193941836259bf"
+DOWNLOAD="https://ufpr.dl.sourceforge.net/project/scidavis/SciDAVis/2/2.4/scidavis-2.4.0.tar.gz"
+MD5SUM="8846e90fb8b75d28c76ba9af2523c189"
 DOWNLOAD_x86_64=""
 MD5SUM_x86_64=""
 REQUIRES="muParser qwt5 qwtplot3d"
```

Comment:
I forgot to change the REQUIRES field from
REQUIRES="muParser qwt5 qwtplot3d"
to
REQUIRES="muParser"

Reply:
Fixed